sequence_tagging_for_ner/README_en.md

```diff
@@ -0,0 +1,173 @@
+
+sequence tagging could be classified into 3 categories, namely Sequence Classification, Segment Classification and Temporal Classification[1]. In this example, only Segment Classification is considered. That is, giving each element in the input sequence a corresponding label in the output sequence. BIO tagging method could indentify entities boundearies, it was widely used. Here is a example of BIO tagging.
+
+<p align="center">
+<img src="images/BIO tag example.png" width="80%" align="center"/><br/>
+fig.1  BIO tagging example
+</p>
+
+According to the sequence tagging results, the entity boundary and the entity category can be obtained directly. Similarly, word segmentation, part of speech tagging, block recognition and semantic role tagging can be solved by sequence tagging.
+
+When using neural network  to solve  problem,  the frontal layers of the network do feature representation learning with input, the last layer of netwok solve the final task based on the learned features. For sequence tagging problem, Recurrent Neural Networs (RNN) are usually used to learn representation features. After that a CRF layer will do the tagging job.
+In fact, the linear part in tradiontal  CRF model is replaced by a nonlinear neural network. The main reason for the use of CRF is that it can better solve the label bias problem by using the likelihood probability of sentence level[2].
+This example is built on the above ideas. Although the NER task is used here as an example, the given model can be applied to a variety of other sequence tagging tasks.
+
+Because of the wide use of sequence tagging, CRF and other classical sequence models are generated. Most of these models only use local information or need human design features. With the development of deep learning research, Recurrent Neural Network (RNN) and other sequential models can deal with the problem of correlation between sequence elements, and can learn feature representation from original input text, which is more suitable for sequence tagging task.
+
+## Model Details
+
+Input of NER is a sentence,  model is going to identify the entities boundaries and categoris. As in [2], only for some simple preprocessing of the original sentence are needed: each word is converted to lowercase, weather it is a uppercase word is a input feature. As shown in Figure 2, the workflow is:
+
+1. yeild input
+ - use one-hot method to represent a sentence
+ - use one-hot method to represent wheather the word is uppercase
+2. look up a tabel, tansfor every word to vector
+3. word vectors are inputed into bi-directonal RNN, to learn new feature repersentation
+4. he learned featur are feed into CRF larer, and its output is sequence tags
```

Comment:
he-> the?
featur-> feature
larer->later?
don't quite get what "sequence tags" is, can you give a further description?